### DISCUSSION TITLE
Unknown client during using wechat login on demo-app.web3auth.io

### DISCUSSION THREAD

**Discussion by phken91** [Web3Auth/Web3Auth#553] [Help]
Hi, we are testing the web3auth with wechat login, since this morning, after we click wechat login will only getting `something went wrong` error.

<img width="1199" alt="image" src="https://user-images.githubusercontent.com/8167012/188559066-4b463c97-1a56-4deb-9568-455db6ff0979.png">

This is the error track code from this page: `39b7ce25328a78ee96b1`
and also it show this error on the page: `invalid_request: Unknown client: cewDD3i6F1vtHeV1KIbaxUZ8vJQjJZ8V`

**Comment by girishvi** [comment]
@himanshuchawla009 Please assist

**Comment by phken91** [comment]
some update, it's also happen on official [demo](https://demo-app.web3auth.io)

**Comment by yashovardhan** [comment]
Hey @lordzpipo & @girishvi indeed it looks like an outage. I've informed the development team. We're looking into this issue.

**Reply by shahbaz17** [reply]
Can you update the status here @yashovardhan ?

Also, @phken91 @girishvi were you able to get this working?

**Comment by chaitanyapotti** [comment]
This was only a problem due to misconfiguration on our end for testnet. This is fixed now. Thanks for reporting
